documentation/docs/09-seo.md

```diff
@@ -0,0 +1,70 @@
+---
+title: SEO
+---
+
+The most important aspect of SEO is to create high-quality content that is widely linked to from around the web. However, there are a few technical considerations for building sites that rank well.
+
+### Zero config SEO benefits
+
+#### SSR
+
+By default, SvelteKit does server-side rendering. While some search engines can index client-side content, not all do. For those that do, there will often be a larger delay between publishing content and having it indexed with client-side rendering compared to server-side rendering. Thus, it is recommended that you leave SSR on for best results. SvelteKit offers highly configurable rendering that will allow you to do things like [dynamic rendering](https://developers.google.com/search/docs/advanced/javascript/dynamic-rendering) where you serve server-side rendered pages to search engines and client-rendered pages to your users. This is not necessarily something that we recommend for a number of reasons such as the increased possibility of having breakages visible only to search engines, the difficulty in creating an exhaustive whitelist of search engine user agents, and a worse experience for users with JavaScript disabled. However, the fact that it would be fairly easy to setup does show off SvelteKit's abilities rather well.
+
+#### Performance
+
+Search engines take site performance into account as a ranking factor. [Web Vitals](https://web.dev/vitals/) is a term coined by Google to provide performance metrics regarding a site's load speed. [Google takes the Core Web Vitals into account](https://developers.google.com/search/blog/2020/11/timing-for-page-experience) in deciding which sites to show at the top of the rankings.
+
+SvelteKit is optimized to give great scores out-of-the-box. Check out [how this site performs on Google PageSpeed Insights](https://pagespeed.web.dev/report?url=https%3A%2F%2Fkit.svelte.dev%2F). We invite you to compare against sites built with other frameworks. If scores on your site are not high, we recommend trying to minimize the size of included dependencies and to use the developer tools provided by your browser to diagnose.
+
+#### Canonicalized URLs
+
+SvelteKit redirects URLs with trailing slashes to versions without trailing slash or vice versa depending on your [configuration](configuration#trailingslash).
+
+### SEO setup
+
+#### `meta` tags
+
+It is generally recommended to add `meta` HTML tags to your page. These can be created just as any other Svelte element. The Svelte Society website has an [example](https://github.com/svelte-society/sveltesociety.dev/blob/staging/src/lib/components/Seo.svelte). You may also find the [`stuff`](loading#input-stuff) feature as helpful for constructing `meta` tags in some cases where you want to pass content from a page to a layout.
```

Comment:
I'd be wary of linking to specific files on an external repo, certainly without also referring to a specific commit. Is there anything else we can do here instead of this?